src/components/structures/IndicatorScrollbar.tsx

```diff
@@ -14,65 +14,59 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import React from "react";
-import PropTypes from "prop-types";
+import React, { createRef } from "react";
 import AutoHideScrollbar from "./AutoHideScrollbar";
 import { replaceableComponent } from "../../utils/replaceableComponent";
 
+interface IProps {
+    // If true, the scrollbar will append mx_IndicatorScrollbar_leftOverflowIndicator
+    // and mx_IndicatorScrollbar_rightOverflowIndicator elements to the list for positioning
+    // by the parent element.
+    trackHorizontalOverflow?: boolean;
+
+    // If true, when the user tries to use their mouse wheel in the component it will
+    // scroll horizontally rather than vertically. This should only be used on components
+    // with no vertical scroll opportunity.
+    verticalScrollsHorizontally?: boolean;
+
+    children: JSX.Element | JSX.Element[];
```

Comment:
think `ReactNode` works here

Reply:
The issue with doing that is that `componentDidUpdate` won't work then I assume, or is there a way to fix this?

Reply:
why wouldn't it work? this is just a build time typing, has no effect on runtime

Reply:
Sorry, I meant it would make TS unhappy but apparently there is `React.Children.count()` to do this